packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4148,6 +4148,43 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       : TextPosition(offset: textBoundary.getLeadingTextBoundaryAt(caretOffset) ?? 0);
   }
 
+  /// Select a paragraph around the given position.
+  ///
+  /// A paragraph boundary is defined as the range that most closely
+  /// encapsulates the given position within two line terminators. If a
+  /// line terminator does not exist in a given direction the selection
+  /// extends to the start/end of the document in that direction.
+  void selectParagraph({required Offset position, SelectionChangedCause? cause}) {
+    selectParagraphsInRange(from: position, cause: cause);
+  }
+
+  /// Selects the set of paragraphs in a document that intersect a given range of global positions.
+  ///
+  /// The set of paragraphs selected are not strictly bounded by the range of global positions.
```

Comment:
I meant that a portion of the selected paragraph might not be entirely within the bounds of given range of global position, but if it intersects with any of the points then it will be selected.